.gitignore

```diff
@@ -66,3 +66,7 @@ compile_commands.json
 
 # Exclude documentation generated by Doxygen
 /doxygen/
+
+# Ignore nix-build result links
+result*
```

Comment:
Why the `*`?

Reply:
Because I usually do something like:
nix-build shell.nix --arg releaseMode true --arg defaultLv2Plugins true --out-link result-$(git branch --show-current)